modules/social_features/social_group/modules/social_group_request/social_group_request.module

```diff
@@ -187,8 +187,10 @@ function social_group_request_preprocess_group(&$variables) {
     return;
   }
 
+  $variables['closed_group'] = TRUE;
```

Comment:
@dmytriikaiun after switching to 10.x branch I see this not relevant change - please fix